source/docs/software/advanced-gradlerio/deploy-git-data.rst

```diff
@@ -28,7 +29,34 @@ Then add the following block to `build.gradle` to add `createVersionFile` as a d
     indent       = "  "
   }
 
-You should also add the `BuildConstants.java` file to the repository `.gitignore`:
+To test this run a build of your project by clicking the WPILib logo in the top right of VS Code and searching for "Build Robot Code". Once the build has finished running there should be a file called ``BuildConstants.java`` in your Java folder. The following is an example of what this file should look like:
```

Comment:
There is a run-on sentence and the sentence doesn't read well. Can you rephrase it?